src/util/RealmClient.js

```diff
@@ -0,0 +1,32 @@
+const Realm = require('realm-web');
+const { realmAppId, realmApiKey } = require('./environmentVariables');
+
+// Connect to your MongoDB Realm app
+const app = new Realm.App(realmAppId);
```

Comment:
This pattern is pulled from [here.](https://docs.mongodb.com/realm/web/graphql-apollo-react/)

Reply:
Can probably add a comment as well so I can find this again.

Reply:
Added.